flask_appbuilder/menu.py

```diff
@@ -41,11 +41,12 @@ def __repr__(self):
 class Menu(object):
     menu = None
 
-    def __init__(self, reverse=True, extra_classes=""):
+    def __init__(self, reverse=True, extra_classes="", security_manager=None):
         self.menu = []
         if reverse:
             extra_classes = extra_classes + "navbar-inverse"
         self.extra_classes = extra_classes
+        self.security_manager = security_manager
```

Comment:
not needed, if we are calling get_data on a request context we can access the security manager with `current_app.appbuilder.sm`